db/migrate/20240813141845_rename_centos_stream_mirror_to_centos.rb

```diff
@@ -0,0 +1,9 @@
+class RenameCentosStreamMirrorToCentos < ActiveRecord::Migration[6.1]
+  def up
+    Medium.unscoped.where(name: "CentOS Stream 9 mirror").update_all(name: "CentOS mirror")
```

Comment:
Along with this media, should we consider updating all other CentOS mirrors which we are removing in this migration, such as `CentOS 7 mirror` and `CentOS Stream`?

Reply:
It doesn't remove old media because users may have reused that and pointed it to vault.centos.org.

I suppose users could have introduced one named `CentOS mirror` which may conflict. I'm not sure what's best here.

Reply:
How about search by URL and then removing those:

```ruby
Medium.unscoped.where(url: "http://mirror.centos.org/centos/$major/os/$arch")
```

Reply:
I don't like to surprise people and manage the data too much. I think I'll go with error handling where it ignores the rename if `CentOS mirror` already exists and add an upgrade warning to the release notes.

Reply:
Confusingly enough, the name is not a unique column so the DB allows multiple entries with the same name. I've come up with a solution that I isn't very pretty, but probably reliable.

I'd appreciate your thoughts on it.

Reply:
The new migration logic makes sense to me.